device/arista/x86_64-arista_7050_qx32/platform_asic

```diff
@@ -0,0 +1 @@
+broadcom
```

Comment:
this platform is only available for sonic-broadcom.swi image, not sonic-broadcom.bin image, can you differentiate that?

Reply:
They are using the same CONFIGURED_PLATFORM (broadcom). So I cannot differentiate them by this dimention. If we have ground truth about bootloader for each platform, we could improve in future.

Reply:
this is a must fix

Reply:
please sync with ying.